File: nordeus_challenge_backend_spring_boot/src/main/java/com/mmileticc/nordeuschallenge/model/dto/RunStartEncounterResponseDTO.java
package com.mmileticc.nordeuschallenge.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RunStartEncounterResponseDTO {

    private RunConfigDTO run;
    private BattleSessionDTO battleSession;
}

File: Gauntlet of the Hero - GOTH/Assets/_Scripts/Models/Backend/RunStartEncounterResponseDto.cs
using Newtonsoft.Json;

namespace GauntletOfTheHero.Backend.Models
{
    [System.Serializable]
    public class RunStartEncounterResponseDto
    {
        [JsonProperty("run")]
        public RunConfigDto run;

        [JsonProperty("battleSession")]
        public BattleSessionDto battleSession;
    }
}